ai_oca_bridge/models/ai_bridge.py

```diff
@@ -32,20 +35,40 @@ class AiBridge(models.Model):
     name = fields.Char(required=True, translate=True)
     active = fields.Boolean(default=True)
     description = fields.Html(translate=True)
-    model_id = fields.Many2one(
-        "ir.model",
-        string="Model",
-        domain=[("transient", "=", False)],
+    user_id = fields.Many2one(
+        "res.users",
+        default=lambda self: self.env.user,
+        help="The user that will be shown when executing this AI bridge.",
+    )
+    payload_type = fields.Selection(
+        [
+            ("record", "Record"),
+            ("record_v0", "Record v0"),  # Deprecated, use 'record' instead
```

Comment:
Delete?

Reply:
I did not delete it because I don't want to break anything if someone installed it before.

As we made it as Beta, we shouldn't do any breaking changes.